diff --git a/node_modules/@react-native-community/blur/android/build.gradle b/node_modules/@react-native-community/blur/android/build.gradle
index 8177235..7b7a919 100644
--- a/node_modules/@react-native-community/blur/android/build.gradle
+++ b/node_modules/@react-native-community/blur/android/build.gradle
@@ -9,7 +9,6 @@ buildscript {
     if (project == rootProject) {
         repositories {
             google()
-            jcenter()
         }
 
         dependencies {
@@ -37,11 +36,11 @@ android {
 
 repositories {
     google()
-    jcenter()
+    maven { url 'https://jitpack.io' }
 }
 
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.eightbitlab:blurview:1.6.3'
+    implementation 'com.github.Dimezis:BlurView:version-1.6.6'
 }
diff --git a/node_modules/@react-native-community/blur/react-native-blur.podspec b/node_modules/@react-native-community/blur/react-native-blur.podspec
index 837cf0c..2e28950 100644
--- a/node_modules/@react-native-community/blur/react-native-blur.podspec
+++ b/node_modules/@react-native-community/blur/react-native-blur.podspec
@@ -10,5 +10,5 @@ Pod::Spec.new do |s|
   s.homepage      = "https://github.com/react-native-community/react-native-blur"
   s.source        = { :git => "https://github.com/react-native-community/react-native-blur.git" }
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
