diff --git a/node_modules/react-native-webview/react-native-webview.podspec b/node_modules/react-native-webview/react-native-webview.podspec
index d0308a1..82538c4 100644
--- a/node_modules/react-native-webview/react-native-webview.podspec
+++ b/node_modules/react-native-webview/react-native-webview.podspec
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/react-native-community/react-native-webview.git", :tag => "v#{s.version}" }
   s.source_files  = "apple/**/*.{h,m}"
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
