diff --git a/node_modules/react-native-shake/react-native-shake.podspec b/node_modules/react-native-shake/react-native-shake.podspec
index 6978674..bed8183 100644
--- a/node_modules/react-native-shake/react-native-shake.podspec
+++ b/node_modules/react-native-shake/react-native-shake.podspec
@@ -12,6 +12,6 @@ Pod::Spec.new do |s|
   s.platform       = :ios, "9.0"
   s.source         = { :git => package_json["repository"]["url"].gsub(/(http.*)/).first, :tag => "v#{s.version}" }
   s.source_files   = 'ios/**/*.{h,m}'
-  s.dependency 'React'
+  s.dependency 'React-Core'
   
 end
