diff --git a/node_modules/react-native-snackbar/RNSnackbar.podspec b/node_modules/react-native-snackbar/RNSnackbar.podspec
index 33fb2eb..9f5f5af 100644
--- a/node_modules/react-native-snackbar/RNSnackbar.podspec
+++ b/node_modules/react-native-snackbar/RNSnackbar.podspec
@@ -16,5 +16,5 @@ Pod::Spec.new do |s|
   s.source_files        = 'ios/**/*.{h,m}'
   s.exclude_files       = 'android/**/*'
   s.exclude_files       = 'example/**/*'
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
