diff --git a/node_modules/react-native-pbkdf2/react-native-pbkdf2.podspec b/node_modules/react-native-pbkdf2/react-native-pbkdf2.podspec
index 447c0df..790382e 100644
--- a/node_modules/react-native-pbkdf2/react-native-pbkdf2.podspec
+++ b/node_modules/react-native-pbkdf2/react-native-pbkdf2.podspec
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/evernym/react-native-pbkdf2.git", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m}"
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
