diff --git a/node_modules/react-native-zip-archive/RNZipArchive.podspec b/node_modules/react-native-zip-archive/RNZipArchive.podspec
index 81bf5af..b0a8e82 100644
--- a/node_modules/react-native-zip-archive/RNZipArchive.podspec
+++ b/node_modules/react-native-zip-archive/RNZipArchive.podspec
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.platform       = :ios, '8.0'
   s.preserve_paths = '*.js'
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
   s.dependency 'SSZipArchive', '2.2.2'
   s.compiler_flags = '-GCC_PREPROCESSOR_DEFINITIONS="HAVE_INTTYPES_H HAVE_PKCRYPT HAVE_STDINT_H HAVE_WZAES HAVE_ZLIB MZ_ZIP_NO_SIGNING $(inherited)"'
 
